pom.xml

```diff
@@ -187,6 +187,28 @@
       <version>3.4</version>
       <scope>test</scope>
     </dependency>
+	<!-- https://mvnrepository.com/artifact/org.powermock/powermock-core -->
```

Comment:
Did you use powermock because of some specific feature that you're missing in easymock?

Reply:
yeah, it's needed to mock the static method.  PS don't merge this yet, working on another take at creating the tracks.